test/Interop/C/modules/repro.swift

```diff
@@ -0,0 +1,5 @@
+// RUN: %empty-directory(%t)
```

Comment:
Is there a better name than "repro" that we can use? Similarly, for other headers in this example. My best suggestion is to actually create a separate subdirectory just for this test since it requires lots of different input files, and their names don't make it obvious that they are related, that they create a particular test case interaction, and shouldn't be reused for other reasons.

Reply:
Makes sense. I've gone with the slightly shorter name `print-qualified-clang-types` -- WDYT?